pkg/rtc/transport.go

```diff
@@ -948,10 +955,16 @@ func (t *PCTransport) AddTrack(trackLocal webrtc.TrackLocal, params types.AddTra
 	if trackLocal.Kind() == webrtc.RTPCodecTypeAudio {
 		configureAudioTransceiver(transceiver, params.Stereo, !params.Red || !t.params.ClientInfo.SupportsAudioRED())
 	}
+	configureTransceiverCodecs(transceiver, enabledCodecs, rtcpFeedbackConfig, !t.params.IsOfferer)
```

Comment:
Made a note in couple of places about subscribed track filtering enabled codecs and setting codec preferences. This is where it happens when a track is added to the peer connection. The preferences are set on the corresponding transceiver.